code/game/objects/items/stacks/knife.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/stack/throwing_knife
+	name = "throwing knife"
+	gender = NEUTER
+	desc = "A knife that is specially designed and weighted so that the wielder\'s strength can be accounted when being thrown."
```

Comment:
>Bent joonk
>Specially designed and weighted

For real?

Reply:
Why, yes? You bent it with CARE :)

Reply:
logically speaking, rebar, too, is a specially designed and weighted piece of metal, made for crowbarring stuff.